stages/org.osbuild.rpm

```diff
@@ -113,7 +113,7 @@ def remove_unowned_etc_kernel(tree, rpm_args):
                           "-qf", "/etc/kernel"],
                          stdout=subprocess.PIPE,
                          check=False)
-    owner = res.stdout.strip()
+    owner = res.stdout.strip().decode()
```

Comment:
(nitpick) by using `subprocess.run(..., text=True)` thisdecode could be avoided I think (or `universal_newlines` if we need to keep compat with really old python)

Reply:
oh, that was an accidental inclusion.  I saw a linter warning and changed it but wanted to do it in a separate commit.

Reply:
Set `encoding="utf-8"`